databricks/MLproject

```diff
@@ -0,0 +1,11 @@
+name: cuML RF test
```

Comment:
Let's move to a subdir (mlflow_project) per discussion.
Actually, if this can all be done without databricks at all, we could move to a top-level dir `cloud-ml-examples/mlflow`